src/main/java/com/feed_the_beast/ftbu/FTBUNotifications.java

```diff
@@ -27,7 +27,10 @@
     public static final INotification CANT_MODIFY_CHUNK = create("cant_modify_chunk", 0).setError(new TextComponentString("Can't modify this chunk!"));
 
     @RegistryObject
-    public static final INotification CANT_CLAIM_CHUNK = create("cant_claim_chunk", 0).setError(new TextComponentString("Claiming is not enabled on this server!"));
+    public static final INotification CLAIMING_NOT_ENABLED = create("cant_claim_chunk", 0).setError(new TextComponentString("Claiming is not enabled on this server!"));
+
+    @RegistryObject
+    public static final INotification CLAIMING_NOT_ALLOWED = create("not_allowed_to_claim_chunk", 0).setError(new TextComponentString("You are not allowed to claim this chunk"));
```

Comment:
This notification's ID could be "cant_claim_chunk" also, and its variant (what currently is 0) could be 1. There's barely any difference, but I prefer the same "group" notifications to have the same ID but different variants